src/code-examples/dialog/html.js

```diff
@@ -1,4 +1,4 @@
-const html = `<button type="button" class="js-open-modal" aria-expanded="true" aria-controls="dialog-content">Open a Dialog</button>
+const html = `<button type="button" class="js-open-modal" aria-controls="dialog-content">Open a Dialog</button>
```

Comment:
Small quibble, but would it make sense to have `aria-expanded="false"` set initially? I know the JS handles setting it, but it might be useful for the example code to demonstrate ARIA usage.

Reply:
I only know this, because I was assigned this dialog component, but we hardcoded it to `true` just in case people didn't have JS enabled. Then immediately used JS to set it to false.

Reply:
If JS isn't enabled this code sandbox wouldn't load at all, right? So I think adding in the `aria-expanded="false"` initially is probably the way to go

Reply:
Oooooo probably! Didn't even think of that.